docs/pages/push-notifications/fcm-credentials.mdx

```diff
@@ -94,7 +94,7 @@ Upload the JSON file to EAS and configure it for sending Android notifications.
 
 <Step label="5">
 
-Configure the **google-services.json** file in your project. Download it from the Firebase Console and place it at the root of your project directory.
+Configure the **google-services.json** file in your project. Download it from the Firebase Console and place it at the root of your project directory. If you're using version control, add it to your ignore file (for example, **.gitignore**) as it contains sensitive data.
```

Comment:
This is already described as a callout in step 4 under the EAS CLI tab. Maybe that can be removed to avoid duplicating this instruction?

Reply:
Got it, thanks for the clarification.